suite/tests/common/nativeexec_exenative.expect

```diff
@@ -0,0 +1,17 @@
+Not under DR!
+calling via IAT-style call
+nativeexec.dll:import_me1(57)
+calling via PLT-style call
+nativeexec.dll:import_me2(37)
+calling via funky ind call
+nativeexec.dll:import_me3(17)
+calling nested native
+nativeexec.exe:print_int(42)
+calling cross-module unwinder
+before longjmp, 1
+after longjmp
+calling indirect ret_imm
+ -> 40
+calling loop_test
+Not under DR!
+all done
```

Comment:
These tests have the entire exe run natively.  I think there is a library here?  It's supposed to be under DR?  I can try to link the annotations into the library.